test/errors-spec.js

```diff
@@ -224,32 +224,42 @@ describe("Parser.expect calls:", function() {
         });
         // Can't test for the [ of an optional group since it's optional
         it("complains about missing } for color", function() {
-            expect("\\color{#ffffff {text}").toFailWithParseError(
-                   "Expected '}', got '{' at position 16:" +
-                   " color{#ffffff {̲text}");
+            expect("\\color{#ffffff{text}").toFailWithParseError(
+                   "Invalid color: '#ffffff{text' at position 8:" +
```

Comment:
Glad you like this change.

Personally I liked the old one better. Which is why prior to the e3dda607afd87a8e6b62fc4135a8f4d50cbee697 suggested by @cbreeden it would report the invalid color as `'#ffffff{…'`, reporting the error in the same place as the old code did. But if you like the new message, that confirms that the code can stay as it is, with the simplifications from the commit I just mentioned.